src/components/registrationForm/RegistrationForm.js

```diff
@@ -9,19 +11,16 @@ class RegistrationForm extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      username: "",
-      password: "",
-      displayName: ""
+      displayName: "",
+      password: ""
```

Comment:
Do we need to send username to the register component?

Reply:
I'm actually still having a problem with my login so probs not.